client-rpc/src/rpc/wallet_rpc.rs

```diff
@@ -126,20 +128,20 @@ where
         self.client.wallets().map_err(to_rpc_error)
     }
 
-    fn list_public_keys(&self, request: WalletRequest) -> Result<Vec<PublicKey>> {
+    fn list_public_keys(&self, request: WalletRequest) -> Result<BTreeSet<PublicKey>> {
```

Comment:
Will the BTreeSet serializable to JSON as an array?

Reply:
Yes